test/functional/lua/overrides_spec.lua

```diff
@@ -63,3 +67,110 @@ describe('print', function()
     eq('\nT\n', redir_exec([[lua print("T\n")]]))
   end)
 end)
+
+describe('debug.debug', function()
+  before_each(function()
+    screen = Screen.new()
+    screen:attach()
+    screen:set_default_attr_ids({
+      [0] = {bold=true, foreground=255},
+      E = {foreground = Screen.colors.Grey100, background = Screen.colors.Red},
+      cr = {bold = true, foreground = Screen.colors.SeaGreen4},
+    })
+  end)
+  it('works', function()
+    write_file(fname, [[
+      function Test(a)
+        print(a)
+        debug.debug()
+        print(a * 100)
+      end
+    ]])
+    eq('', redir_exec('luafile ' .. fname))
+    feed(':lua Test()\n')
+    screen:expect([[
+      {0:~                                                    }|
+      {0:~                                                    }|
+      {0:~                                                    }|
+      {0:~                                                    }|
+      {0:~                                                    }|
+      {0:~                                                    }|
+      {0:~                                                    }|
+      {0:~                                                    }|
+      {0:~                                                    }|
+      {0:~                                                    }|
+      {0:~                                                    }|
+      {0:~                                                    }|
+      nil                                                  |
+      lua_debug> ^                                          |
+    ]])
```

Comment:
Screen tests are a relatively expensive (that's why 'inccommand' tests take ~30s...), it can save milliseconds per test to use smaller screen. Just mentioning this since the vertical space isn't used in these tests, could chop off some lines.